services/apps/premium/members_enrichment_worker/src/utils/normalize.ts

```diff
@@ -169,18 +169,21 @@ const fillPlatformData = async (
   member: IMember,
   enriched: EnrichmentAPIMember,
 ): Promise<IMember> => {
-  if (enriched.emails && Array.isArray(enriched.emails)) {
-    if (enriched.emails.length > 0) {
-      const emailSet = new Set<string>(
-        enriched.emails.filter((email) => !email.includes('noreply.github')),
-      )
-
-      if (!member.emails) {
-        member.emails = []
-      }
-      member.emails.forEach((email) => emailSet.add(email))
-      member.emails = Array.from(emailSet)
-    }
+  // NOTE: Enrichment API returns a list of email address not associated to the
+  // actual member. We simply ignore these for now.
+  // if (enriched.emails && Array.isArray(enriched.emails)) {
+  //   if (enriched.emails.length > 0) {
+  //     const emailSet = new Set<string>(
+  //       enriched.emails.filter((email) => !email.includes('noreply.github')),
+  //     )
+
+  //     member.emails.forEach((email) => emailSet.add(email))
+  //     member.emails = Array.from(emailSet)
+  //   }
+  // }
+
+  if (!member.emails) {
+    member.emails = []
   }
```

Comment:
I assume this is checking if the member already had emails? Just making sure if we already had them, we are not overwriting.